test/screenshot/checkbox/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './index.scss';
+
+import Checkbox from '../../../packages/checkbox';
```

Comment:
What's the benefit of this? I see that some screenshot tests import `/index` and some don't

Reply:
we should be using /index. it doesn't use the built version in /dist. Its easier to debug and fix issues since we can just update the /index.js file. We also know what were testing against every time (not /dist/index).